### PR TITLE
[Tool] AirByte Connector Fixed TLS support

### PR DESCRIPTION
**Pull request by brmei**
Fixes #issue

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [x] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: contrib/destination-starrocks/build.gradle [hunk 1]
```diff
@@ -15,7 +15,7 @@ dependencies {
     implementation 'com.alibaba:fastjson:1.2.75'
 
     implementation project(':airbyte-config-oss:config-models-oss')
-//    implementation project(':airbyte-protocol:protocol-models')
+    // implementation project(':airbyte-protocol:protocol-models')
```
Comment (chaoyli) [inline contrib/destination-starrocks/build.gradle:18]: Why annotate this protocol?

File: contrib/destination-starrocks/src/main/java/io/airbyte/integrations/destination/starrocks/DefaultStreamLoader.java [hunk 2]
```diff
@@ -153,27 +98,27 @@ public StreamLoadResponse send(List<AirbyteRecordMessage> records) throws Except
...
-        // check if port for approapriate url scheme
-        if (port == 443) {
+        // check if port for appropriate url scheme
+        if (properties.getSSL()) {
             scheme = "https";
         }
 
         String sendUrl = String.format("%s://%s:%d%s", scheme, host, port, loadUrlPath);
-        String label = StreamLoadUtils.label(loadTable);
+        LOG.info("Stream Load URL : {}", sendUrl);
```
Comment (chaoyli) [inline contrib/destination-starrocks/src/main/java/io/airbyte/integrations/destination/starrocks/DefaultStreamLoader.java:107]: You can remove the log in the formal code, just for debugging.

File: contrib/destination-starrocks/src/main/java/io/airbyte/integrations/destination/starrocks/DefaultStreamLoader.java [hunk 3]
```diff
@@ -153,27 +98,27 @@ public StreamLoadResponse send(List<AirbyteRecordMessage> records) throws Except
...
 
+        String label = StreamLoadUtils.label(loadTable);
         HttpPut httpPut = new HttpPut(sendUrl);
-        httpPut.setConfig(RequestConfig.custom().setExpectContinueEnabled(true).setRedirectsEnabled(true).build());
+
         httpPut.setEntity(new StreamLoadEntity(records));
-        httpPut.setHeaders(defaultHeaders);
+        httpPut.setHeaders(defaultHeaders);;
```
Comment (Astralidea) [inline contrib/destination-starrocks/src/main/java/io/airbyte/integrations/destination/starrocks/DefaultStreamLoader.java:113]: repeated semicolon

File: contrib/destination-starrocks/src/test-integration/java/io/airbyte/integrations/destination/starrocks/StarrocksDestinationAcceptanceTest.java [hunk 4]
```diff
@@ -12,6 +12,10 @@
 import io.airbyte.integrations.standardtest.destination.DestinationAcceptanceTest;
 import java.io.IOException;
 import java.nio.file.Paths;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
```
Comment (Astralidea) [inline contrib/destination-starrocks/src/test-integration/java/io/airbyte/integrations/destination/starrocks/StarrocksDestinationAcceptanceTest.java:18]: Unnecessary imports?

File: contrib/destination-starrocks/build.gradle (hunk 1 above)
Comment (brmei) [reply contrib/destination-starrocks/build.gradle:18]: It was already annotated before, I just moved the position of the slashes.

File: contrib/destination-starrocks/src/test-integration/java/io/airbyte/integrations/destination/starrocks/StarrocksDestinationAcceptanceTest.java (hunk 4 above)
Comment (brmei) [reply contrib/destination-starrocks/src/test-integration/java/io/airbyte/integrations/destination/starrocks/StarrocksDestinationAcceptanceTest.java:18]: Imports are used in the file. The code wasn't compiling before.

File: contrib/destination-starrocks/src/main/java/io/airbyte/integrations/destination/starrocks/DefaultStreamLoader.java (hunk 3 above)
Comment (brmei) [reply contrib/destination-starrocks/src/main/java/io/airbyte/integrations/destination/starrocks/DefaultStreamLoader.java]: I removed it now

File: contrib/destination-starrocks/src/main/java/io/airbyte/integrations/destination/starrocks/DefaultStreamLoader.java (hunk 2 above)
Comment (brmei) [reply contrib/destination-starrocks/src/main/java/io/airbyte/integrations/destination/starrocks/DefaultStreamLoader.java]: Alright, it is gone now

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: contrib/destination-starrocks/build.gradle (hunk 1 above)
Comment (chaoyli) [reply contrib/destination-starrocks/build.gradle:18]: You can remove it if not needed.

Comment (brmei) [reply contrib/destination-starrocks/build.gradle]: It has been removed.

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Pull request closed without merging**